internal/static/i18n/en.yaml

```diff
@@ -560,7 +560,7 @@ EventTypes:
     removed: Target deleted
   user:
     added: User added
-    selfregistered: User registered himself
+    selfregistered: User registered
```

Comment:
As we used google translate for most languages, I this we should update all of them to be gender-neutral.